src/docker.go

```diff
@@ -81,10 +81,9 @@ func main() {
 		)
 		exitOnErr(err)
 		var tmpDocker *client.Client
-		tmpDocker, err = client.NewEnvClient()
```

Comment:
It was already there, but `tmpDocker` is needed?

Reply:
It is non-obvious, but `var docker nri.DockerClient` above is a custom interface, and `tmpDocker` is the upstream docker client. SInce the interface does not have the `Close()` method, `tmpDocker` is used to defer that call.

We could add `Close()` to our interface and make it do nothing for the fargate implementation, but might be a bit more convoluted than this workaround. What would you prefer?

Reply:
fine with the current workaround, I would not add more changes to the PR and it would be a bit out of scope